### PR TITLE
Add interactive list

### PR DESCRIPTION
**Pull request by Laylo309**
- Add interactive functionalities
- Add Assets folder
- Create new Javascript files
- Update index.html

### CODE REVIEW

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @Laylooo, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly try to update the `live link` in the README file. Right now it is giving an error.
- In this getting started part there are some instructions that you have missed for a user to use your project. such as `npm install`, and `npm start`. 

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Assets/interaction.js [hunk 1]
```diff
@@ -32,13 +32,11 @@ export default class Interaction {
...
     checklists.addEventListener('change', () => {
       task.completed = !task.completed;
-      //  const descript = document.getElementById(`d${task.index}`);
-      //  descript.classList.toggle('line');
     });
   }
 
   static checkList() {
-    const storage = this.localStorage;
+    const storage = this.localStorageA();
```
Comment (bellom) [inline src/Assets/interaction.js:39]: Oops, we need to address the issue the previous reviewer mentioned.

In other words;

For some reason, our data is not properly saved by the localStorage.

Checked item after reload tends to change to the unchecked item.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Laylooo  👏 , 

 ## Required Changes ♻️
 
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @Laylooo**

### **Project Approved ✅** 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**